Makefile

```diff
@@ -26,19 +26,26 @@ ifeq ($(call vercmp,$(kubernetes_version),gteq,1.25.0), true)
 	ami_component_description ?= (k8s: {{ user `kubernetes_version` }}, containerd: {{ user `containerd_version` }})
 endif
 
-OS=
-ifneq (,$(findstring al2023, $(PACKER_TEMPLATE_FILE)))
-	OS=-al2023
+ifneq ($(origin fips), undefined)
+	enable_fips=true
```

Comment:
I think we only need `enable_fips`, kind of confusing to have both

Reply:
Removed this logic.